### PR TITLE
Use the tilde convention as per Module Standard.

### PR DESCRIPTION
**Pull request by camfindlay**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	],
 	"type": "silverstripe-module",
 	"require": {
-		"silverstripe/cms": ">=3.1.0",
+		"silverstripe/cms": "~3.1",
```
Comment (micmania1) [inline composer.json]: Would it not be better to use `^`? What was the reason `~` was chosen as the standard?

Comment (micmania1) [inline composer.json]: For reference: https://getcomposer.org/doc/articles/versions.md#caret

Comment (assertchris) [inline composer.json]: I would prefer `^` over `~` because it's more inline with modern composer/npm version specification...

Comment (assertchris) [inline composer.json]: I am fairly sure the module standard doesn't say use `~` over `^`. If it does, that wasn't added with group consensus... :D

**Comment by kinglozzer** [comment]
@camfindlay could you incorporate/cherry-pick #298 into this too? :)

**Pull request closed without merging**